nilearn/decoding/searchlight.py

```diff
@@ -74,8 +74,9 @@ def search_light(X, y, estimator, A, groups=None, scoring=None,
 
     Returns
     -------
-    scores : array-like of shape (number of rows in A)
-        search_light scores
+    scores : dict of float of shape=(n_voxels, n_fold)
```

Comment:
I don' understand why you represent a structured set of homogeneous numerical values by a dictionary instead of a Numpy array ?

Reply:
The only reason was that I can easily check each measure by looking only at dictionary key. But I can do a numpy array of (n_measures, n_voxels, n_fold).